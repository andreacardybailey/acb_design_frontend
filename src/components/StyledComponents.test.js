import styled from 'styled-components';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
const Button = styled.button`
  color: red;
`;
describe("Testing Styled Components Menu and Toggle", () => {
  test('Styled Components are rendering', () => {
    const tree = renderer.create(<Button />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})