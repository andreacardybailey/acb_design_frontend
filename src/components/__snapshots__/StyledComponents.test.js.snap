// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Styled Components Menu and Toggle Styled Components are rendering 1`] = `
<button
  className="jeLoYQ"
/>
`;
